src/components/auction/OrderPlacement/index.tsx

```diff
@@ -221,6 +224,44 @@ const OrderPlacement: React.FC<OrderPlacementProps> = (props) => {
     }
   }, [onUserPriceInput, price, derivedAuctionInfo, showPriceInverted])
 
+  useEffect(() => {
```

Comment:
Why do you use useEffect in this situation?

Reply:
It was not necessary!